tests/Integration/data/helpers.php

```diff
@@ -6,3 +6,14 @@ function transformAcceptsNonNullableCallable(): void
 {
     transform(User::first(), fn (User $user) => $user->toArray());
 }
+
+function retryAcceptsCallableAsSleepMilliseconds(): void
+{
+    retry(5, function (int $attempt): bool {
+        return false;
+    }, function (int $attempt, Exception $e): int {
+        return 0;
+    }, function (Exception $e): bool {
```

Comment:
`Exception` doesn't exist in the current namespace:
```suggestion
    }, function (int $attempt, \Exception $e): int {
        return 0;
    }, function (\Exception $e): bool {
```